docs/src/content/useCases.js

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import GradientText from 'docs/src/components/typography/GradientText';
+import ROUTES from '../route';
 
 const useCases = {
   overline: 'Use cases',
```

Comment:
To my mind, even the current description text 
<img width="1279" alt="Screenshot 2023-06-07 at 3 32 27 PM" src="https://github.com/mui/mui-toolpad/assets/19550456/9d4a06ec-d1b9-4ab4-bb1d-db971f805038"> can be too long, so we should probably aim at reducing this instead of adding more

Reply:
There is no buttons yet we'll put them when we'll put all 3 apps on the docs.